.coveragerc

```diff
@@ -228,7 +228,6 @@ omit =
     homeassistant/components/denonavr/__init__.py
     homeassistant/components/denonavr/media_player.py
     homeassistant/components/denonavr/receiver.py
-    homeassistant/components/digital_ocean/*
```

Comment:
The PR does not add test for existing functionality, replace the wildcard with the modules which are still not fully covered, e.g.:
```
homeassistant/components/digital_ocean/__init__.py
homeassistant/components/digital_ocean/binary_sensor.py
homeassistant/components/digital_ocean/switch.py
```

Reply:
Good or not, this is how we choose to do it in this project.
It's of course always warmly welcome to add tests, and when doing so, remove uncovered files from .coveragerc